webpack.config.js

```diff
@@ -14,11 +15,36 @@ module.exports = {
           },
         },
       },
+      {
+        test: /\.s?css$/,
+        use: ['style-loader', 'css-loader', 'sass-loader'],
+      },
     ],
   },
   optimization: {
     minimize: true,
-    minimizer: [new TerserPlugin()],
+    minimizer: [
+      new TerserPlugin({
+        sourceMap: true,
```

Comment:
Should this be enabled for the production config?

Reply:
Fair comment! I'll rip it out, it's not helping anyway.